index.js

```diff
@@ -35,8 +35,12 @@ module.exports = function (archive, target, opts, cb) {
       ignored: opts.ignore
     })
     watcher.once('ready', function () {
-      watcher.on('add', consume)
-      watcher.on('change', consume)
+      watcher.on('add', function (file, stat) {
+        consume(file, noop)
```

Comment:
let's rather call `consume(file)` here, so it will default the callback to `emitError` automatically